### PR TITLE
[improve][broker]correct the semantics of loadBalancerHistoryResourcePercentage

### PR DESCRIPTION
**Pull request by HQebupt**
### Motivation
correct the semantics of loadBalancerHistoryResourcePercentage like `loadBalancerBrokerOverloadedThresholdPercentage` and  `loadBalancerBrokerThresholdShedderPercentage`, 
https://github.com/apache/pulsar/blob/bb0bfb904da90030c171ceac0159142d2bdb51b7/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2026-L2038

### Modifications
correct the semantics of loadBalancerHistoryResourcePercentage 

### Verifying this change

- [x]  Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:
If `yes` was chosen, please highlight the changes

- Dependencies (does it add or upgrade a dependency): (no)
- The public API: (no)
- The schema: (no)
- The default values of configurations: (no)
- The wire protocol: (no)
- The rest endpoints: (no)
- The admin cli options: (no)
- Anything that affects deployment: (no)

### Documentation
Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
- [x] `doc-not-needed`

### CODE REVIEW

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 1]
```diff
@@ -2050,7 +2050,7 @@ public class ServiceConfiguration implements PulsarConfiguration {
             category = CATEGORY_LOAD_BALANCER,
             doc = "Resource history Usage Percentage When adding new resource usage info"
     )
-    private double loadBalancerHistoryResourcePercentage = 0.9;
+    private int loadBalancerHistoryResourcePercentage = 90;
```
Comment (Jason918) [inline pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java:2053]: This change is not backward compatible.

**Review by Jason918 — CHANGES_REQUESTED** [review]

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java (hunk 1 above)
Comment (HQebupt) [reply pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java]: Can we keep the field double type ?

File: conf/broker.conf [hunk 2]
```diff
@@ -1181,7 +1181,7 @@ loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold=4
 
 # When calculating new resource usage, the history usage accounts for.
 # It only takes effect in the ThresholdShedder strategy.
-loadBalancerHistoryResourcePercentage=0.9
+loadBalancerHistoryResourcePercentage=90
```
Comment (Jason918) [inline conf/broker.conf:1184]: You have to consider the case when new version broker using broker.conf in previous version.

Comment (HQebupt) [reply conf/broker.conf:1184]: This change lacks the compatibility. The new version can only use the new configuration file, the old version use the old one.

Comment (Jason918) [reply conf/broker.conf:1184]: This is a break change. We better not do this.

**Pull request closed without merging**